### PR TITLE
GVT test cases

### LINKED ISSUE #27910: Sample GVT Bucket
Sample GVT test cases have been added for the JMX REST Connector APIs.

### PR DESCRIPTION
**Pull request by seethalm**
fixes #27910 

Sample GVT bucket created which includes test cases against JMX rest connector endpoints.

### CODE REVIEW

**Comment by seethalm** [comment]
#build

**Comment by seethalm** [comment]
#build

**Comment by seethalm** [comment]
#build

**Comment by seethalm** [comment]
#build

File: dev/fattest.simplicity/src/componenttest/topology/utils/HttpUtils.java [hunk 1]
```diff
@@ -61,6 +61,8 @@ public class HttpUtils {
 
     private final static int LOWEST_ERROR_CODE = 400;
 
+    public static final String PEER_CERTIFICATES = "PEER_CERTIFICATES";
```
Comment (websterc87) [inline dev/fattest.simplicity/src/componenttest/topology/utils/HttpUtils.java]: Not sure what the point of this change is, now stuff is back in GVT Utils.

**Review by websterc87 — DISMISSED** [review]
Failures in PB appear to be timeout related and pre existing issues.

Minor bit of usused code that could be tidying up.

In general not keen on the move of the code back to GVTUtils as it wont help the long term development of GVT fats. It has worked for now. But I suspect you will want to consider moving it to common code when the documentation about how to write GVT Buckets are being written. For now it works so not blocking delivery.

File: dev/fattest.simplicity/src/componenttest/topology/utils/HttpUtils.java (hunk 1 above)
Comment (seethalm) [reply dev/fattest.simplicity/src/componenttest/topology/utils/HttpUtils.java]: Removed the change

**Review by websterc87 — APPROVED** [review]
Happy with the Peer certificates removal

**Comment by seethalm** [comment]
#run-libby-bot

**Pull request merged**